modules/node-shared/src/main/scala/org/tessellation/node/shared/package.scala

```diff
@@ -32,20 +30,20 @@ package object shared {
 
   val nodeSharedKryoRegistrar: Map[Class[_], KryoRegistrationId[NodeSharedOrSharedRegistrationIdRange]] =
     Map[Class[_], NodeSharedKryoRegistrationId](
-      classOf[PingConsensusHealthStatus] -> 500,
-      classOf[PingHealthCheckKey] -> 501,
-      classOf[HealthCheckRoundId] -> 502,
-      classOf[PeerAvailable] -> 503,
-      classOf[PeerUnavailable] -> 504,
-      classOf[PeerUnknown] -> 505,
+      // classOf[PingConsensusHealthStatus] -> 500,
```

Comment:
since these are registered historically, I assume keeping the comments here is helpful for future development? If that is the case, should we add a comment to not remove past registrations or can the indices be overwritten?

Reply:
I need to remember what we did previously as we've already had such a situation. Probably we keep it commented out, not to reuse the id. Anyway, hopefully we'll get rid of kryo.